editor/import_dock.cpp

```diff
@@ -633,6 +633,10 @@ void ImportDock::_advanced_options() {
 }
 
 void ImportDock::_reimport() {
+	//get the current object in the inspector
+	Object *inspector_current_object = ObjectDB::get_instance(EditorNode::get_singleton()->get_editor_selection_history()->get_current());
```

Comment:
This seems a bit convoluted, we might have an easier way to access the currently edited object (@KoBeWi might know).

Reply:
`InspectorDock::get_inspector_singleton()->get_edited_object()`